Client/Frontend/Browser/Tab.swift

```diff
@@ -76,7 +77,9 @@ class Tab: NSObject {
     var mimeType: String?
     var isEditing: Bool = false
     var shouldClassifyLoadsForAds = true
-
+    var rewards: BraveRewards?
```

Comment:
Does the AppDelegate's rewards object change? This copies the reference with a strong point, would be best to avoid this.

- Can we just pull the rewards object off the app delegate when we need it?
- Or if necessary, potentially make this a getter
- At the very least, this should _probably_ be a `weak` var.